Helper/Order.php

```diff
@@ -1116,6 +1126,17 @@ public function voidTransactionOnBolt($transactionId, $storeId)
             throw new LocalizedException(__('Payment void error.'));
         }
 
+        $payment = $order->getPayment();
+        $transaction = $this->transactionRepository->getByTransactionType(
```

Comment:
Let's put it into catch throw block: if transaction doesn't exist for some reason it's better for us to proceed.